src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -26,48 +26,6 @@ def setUp(self):
 
     def tearDown(self):
         pass
-	
-    def assertCbsdRecordEqual(self, first_cbsd, second_cbsd):
```

Comment:
For what it's worth, I thought this helper function looked useful (it should help keep the code below a bit cleaner). You could have it take in registration_request, grant_request, grant_response, cbsd_record (including the registration and grant data).